components/review/shared/reviewRatingPage.tsx

```diff
@@ -1,23 +1,65 @@
 import { ReviewStage } from "pages/review";
 import { ReviewSplitPanelPage } from "./reviewSplitPanelPage";
+import Button from "@components/common/Button";
 
 interface Props {
   studentName: string;
   currentStage: ReviewStage;
+  currentStageRubric: JSX.Element;
+  currentStageAnswers: JSX.Element;
   title: string;
+  resumeLink?: string;
+  scores: Map<ReviewStage, number>;
 }
 
+interface resumeProps {
+  resumeLink: string;
+}
+
+const ResumeLink: React.FC<resumeProps> = ({ resumeLink }) => {
+  return (
+    <div className="flex flex-col gap-8">
+      <Button
```

Comment:
Can we make this an `a` tag instead, so that the link opening behaviour is built-in?  It should be fairly simple to make the `Button` component support using `a` tags.